client/src/components/Layout/Sidebar.tsx

```diff
@@ -0,0 +1,30 @@
+import { Layout, Button } from 'antd';
+import React from 'react';
+import styles from './Sidebar.module.css';
+
+const { Header, Footer } = Layout;
+
+const Sidebar: React.FC = () => {
+  const userName: string = 'superadmin';
+
+  return (
+    <Layout className={`${styles.sidebar} ${styles['sidebar-layout']}`}>
+      <Header className={`${styles.sidebar} ${styles['sidebar-header']}`}>
+        <img src="/src/assets/wospLogo.png" alt="WOSP Logo" />
+      </Header>
+      <Layout className={`${styles.sidebar} ${styles['sidebar-content']}`}>
+        <a href="">Strona Główna</a>
```

Comment:
Change <a> tags to react-router <NavLink /> tags. Also add detection of currently active page (styles like on hover). Here are the docs: https://reactrouter.com/en/main/components/nav-link